PORTAL/jobs/__main__.py

```diff
@@ -1076,6 +1076,16 @@ def add_cmd(name, subs=subs, *, parents=(), **kwargs):
     return cmd, ns, cfgfile, user, verbosity, logfile, devmode
 
 
+def _should_ensure_next(cmd):
+    # In some cases the mechanism to run jobs from the queue may
+    # get interrupted, so we re-start it manually here if necessary.
+    if cmd in ('queue-info', 'compare'):
+        return False
```

Comment:
Something is broken for `ensure_next()`.

```
$ bench-jobs-dev compare req-compile-bench-1663116583-esnow req-compile-bench-1663116597-esnow
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 192, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/esnow/faster-cpython-tools/PORTAL/jobs/__main__.py", line 1177, in <module>
    main(cmd, cmd_kwargs, cfgfile, user, devmode)
  File "/home/esnow/faster-cpython-tools/PORTAL/jobs/__main__.py", line 1126, in main
    jobs.ensure_next()
  File "/home/esnow/faster-cpython-tools/PORTAL/jobs/__init__.py", line 272, in ensure_next
    queue = self.queue.snapshot
  File "/home/esnow/faster-cpython-tools/PORTAL/jobs/queue.py", line 246, in snapshot
    assert pid is not None
AssertionError
```

I'll open a separate issue for this.  In the meantime this workaround should unblock me.